src/sgd/tools/blast/__init__.py

```diff
@@ -169,9 +169,11 @@ def _get_blast_options(p):
     if filter == 'on':
         if program != 'blastn':
             options = options + " -seg yes"
-        else:
-            options = options + " -dust '20 64 1'"
```

Comment:
why do we have to remove this else condition?

Reply:
For blastn, it turns on the filter by default.

Reply:
oh..ok